lib/registry.ts

```diff
@@ -64,7 +70,7 @@ export function getAppTranslations(appId: string): AppTranslations {
 		typeof window._oc_l10n_registry_translations === 'undefined'
 		|| typeof window._oc_l10n_registry_plural_functions === 'undefined'
 	) {
-		console.warn('No OC L10N registry found')
+		console.warn(`No translation for ${appId} found`)
```

Comment:
I think this warning is a bit misleading (while it is true that there are no translations, this warning means there is no registry available).

Reply:
Yep, correct.

Maybe rewrite it to `"No translation has been registered"`?

Or even add a second check for `window._oc_l10n_registry_translations[appId]` and warn `No translation for appId "${appId}" has been registered`?

Reply:
> Maybe rewrite it to "No translation has been registered"?

I think this would be sufficient :)

Reply:
Too overhead or ok? :)

```ts
export function getAppTranslations(appId: string): AppTranslations {
	if (
		typeof window._oc_l10n_registry_translations === 'undefined'
		|| typeof window._oc_l10n_registry_plural_functions === 'undefined'
	) {
		console.warn('No translations has been registered')
		return {
			translations: {},
			pluralFunction: (number: number) => number,
		}
	}

	if (typeof window._oc_l10n_registry_translations[appId] === 'undefined') {
		console.warn(`No translation for appId "${appId}" have been registered`)
	}

	return {
		translations: window._oc_l10n_registry_translations[appId] || {},
		pluralFunction: window._oc_l10n_registry_plural_functions[appId],
	}
}
```